lighthouse-core/audits/non-composited-animations.js

```diff
@@ -27,6 +27,16 @@ const UIStrings = {
     =1 {Unsupported CSS Property: {properties}}
     other {Unsupported CSS Properties: {properties}}
   }`,
+  /** Name of a compositor failure reason where the transform being animated depends on the box size. */
```

Comment:
I imagine these might be rough for the translators without more context, maybe something more like the below for these?
```suggestion
  /** Descriptive reason for why a user-provided animation failed to be optimized by the browser due to a `transform` property being dependent on the size of the element itself. Shown in a table with a list of other potential failure reasons.  */
```

also a decent gut check for whether I understood the reason :)

Reply:
also probably a very compelling case for doing really comprehensive web.dev documentation on examples of these reasons and how to fix them